olive/cli/auto_opt.py

```diff
@@ -327,19 +354,23 @@ def _get_passes_config(self, passes_config: Dict[str, Any], olive_config: OliveP
     },
     "passes": OrderedDict(
         [
+            ("capture_split_info", {"type": "CaptureSplitInfo"}),
             ("conversion", {"type": "OnnxConversion"}),
             ("model_builder", {"type": "ModelBuilder", "precision": "fp32", "metadata_only": False}),
-            ("transformer_optimizer", {"type": "OrtTransformersOptimization"}),
+            (
+                "transformer_optimizer",
+                {"type": "OrtTransformersOptimization", "opt_level": 0, "use_fp16": False, "keep_io_types": False},
```

Comment:
why not do opt_level 99?

Reply:
opt_level>0 applies ort runtime graph optimizations to the model graph but this is not required since it can be done during session loading. This type of graph optimization has multiple rules such as operator fusion (matmul + add, etc) and renaming nodes that might break downstream passes. So, I turned it off like we do in most of our examples.